metadata.yaml

```
@@ -0,0 +1,10 @@
+name: glauth-operator
```

Should the name just `glauth`?

```suggestion
name: glauth
```

---

Yes, updated!